keps/2145-list-admitted-workloads/README.md

```diff
@@ -0,0 +1,145 @@
+# KEP-1834: List Admitted And Not Finished Workloads
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+Add a new visibility endpoint in Kueue for querying the list of Workloads that are still "running".
+
+## Motivation
+
+Jsonpath support in kubectl is limited, and we can not filter resources by condition in this way. By adding a new
+visibility endpoint, users can list running workloads by `kubectl get`.
```

Comment:
This functionality is already being worked on in kueuectl list workloads (see: https://github.com/kubernetes-sigs/kueue/blob/8d7f709225bb828c38758a22ccf892f304b1bd0c/keps/2076-kueuectl/README.md#list-workloads 
and 
https://github.com/kubernetes-sigs/kueue/pull/2195). Is there another reason why we might need this extra api?

Reply:
@KunWuLuan before we go further with this PR could you please address that?